.github/scripts/common.sh

```
@@ -0,0 +1,2358 @@
+  # https://github.com/grpc/grpc/blob/master/doc/fork_support.md
+  export GRPC_ENABLE_FORK_SUPPORT=0
+
+  local eloqkv_install_path="/home/$current_user/workspace/eloqkv/install"
+
+  if [ ! -d "/home/$current_user/workspace/eloq_test/" ]; then
+    echo "/home/$current_user/workspace/eloq_test/ not exists, exit !!!"
+  fi
+
+  setup_passwordless_ssh_for_eloq_test
+
+  cd /home/$current_user/workspace/eloq_test
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)